src/platform/packages/shared/kbn-scout/src/playwright/fixtures/worker/core_fixtures.ts

```diff
@@ -56,16 +57,25 @@ export const coreWorkerFixtures = base.extend<
     esClient: Client;
     kbnClient: KbnClient;
     samlAuth: SamlAuth;
+    flags: Flags;
   }
 >({
   // Provides a scoped logger instance for each worker to use in fixtures and tests.
   // For parallel workers logger context is matching worker index+1, e.g. '[scout-worker-1]', '[scout-worker-2]', etc.
+  flags: [
+    ({}, use) => {
+      const flags = getFlags(process.argv.slice(2));
+      use(flags);
+    },
+    { scope: 'worker' },
+  ],
   log: [
-    ({}, use, workerInfo) => {
+    ({ flags }, use, workerInfo) => {
       const workersCount = workerInfo.config.workers;
       const loggerContext =
         workersCount === 1 ? 'scout-worker' : `scout-worker-${workerInfo.parallelIndex + 1}`;
-      use(getLogger(loggerContext));
+
+      use(new ScoutLogger(loggerContext, pickLevelFromFlags(flags)));
```

Comment:
@dmlemeshko Let me know what you think - it doesn't seem like the global getLogger() is needed anymore, and this allows us to control the logging level centrally.

Reply:
I agree, central control is a much better approach. With this change I don't see Scout debug logs in your PR pipeline, so I think we need to add `--debug` for scout script in https://github.com/elastic/kibana/blob/9200fd8dcf79d02f544aeb5b91505a25a111a120/.buildkite/scripts/steps/test/scout_configs.sh#L85

It is still useful to better understand what takes time (e.g. creating Kibana Space or importing Saved Objects)

Reply:
I don't think that `scripts/scout` respects `--debug` flag with this change: can't find any `log.debug` messages

Reply:
>yes, this defaults to info. I can also default to debug, but if we just need it for CI it's better to set it to debug explicitly there I think?

That is what I tried to do, but it is not reaching the Playwright runner:

command run: `node scripts/scout.js run-tests --serverless=oblt --config x-pack/platform/plugins/private/discover_enhanced/test/scout/ui/parallel.playwright.config.ts --debug`

```
proc [playwright] Running tests with flags: {"verbose":false,"quiet":false,"silent":false,"debug":false,"help":false,"_":[],"unexpected":[],"v":false}
```